### PR TITLE
chore(deps): Update openapi-extractor to v1.3.0

### PR DESCRIPTION
**Pull request by provokateurin**
### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included or not possible
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

**Comment by provokateurin** [comment]
Fun, psalm locally doesn't produce the error

**Comment by nickvergessen** [comment]
> Fun, psalm locally doesn't produce the error

It does for me, but dropping 073096c570e63414aa587f80b794f775febe5324 should work? It claims to drop an unused schema and its used but you replace it with something?

**Comment by nickvergessen** [comment]
Ah, it's not used in the OpenAPI, but only when doing commands.

File: lib/Model/BotServer.php [hunk 1]
```diff
@@ -62,7 +59,19 @@ public function __construct() {
 	}
 
 	/**
-	 * @return TalkBotWithDetailsAndSecret
+	 * @return array{
```
Comment (nickvergessen) [inline lib/Model/BotServer.php:62]: booo :mega:

**Review by nickvergessen — APPROVED** [review]

File: lib/Model/BotServer.php (hunk 1 above)
Comment (provokateurin) [reply lib/Model/BotServer.php:62]: I knowwww :pensive:

**Pull request merged**